pkg/sql/codegen/attribute/checker.go

```diff
@@ -100,6 +100,28 @@ func IntRangeChecker(lower, upper *int, includeLower, includeUpper bool) func(in
 	return checker
 }
 
+// BoolIntChecker returns a checker function that do **not** check the input.
```

Comment:
The comment is not inconsistent with the following code, BoolIntChecker **do** check the input.